pandas/core/frame.py

```diff
@@ -3484,16 +3484,13 @@ def _ixs(self, i: int, axis: int = 0):
         """
         # irow
         if axis == 0:
-            new_values = self._mgr.fast_xs(i)
+            new_mgr = self._mgr.fast_xs(i)
 
             # if we are a copy, mark as such
-            copy = isinstance(new_values, np.ndarray) and new_values.base is None
-            result = self._constructor_sliced(
-                new_values,
-                index=self.columns,
-                name=self.index[i],
-                dtype=new_values.dtype,
-            ).__finalize__(self)
+            copy = isinstance(new_mgr.array, np.ndarray) and new_mgr.array.base is None
```

Comment:
not for this PR, but this line is something we should handle more systematically, will be wrong for NDArrayBackedEAs